examples/player/player_wrapper.rs

```diff
@@ -16,12 +17,37 @@ use std::thread::Builder;
 pub struct PlayerWrapper {
     player: Arc<Mutex<Box<Player>>>,
     shutdown: Arc<AtomicBool>,
+    use_gl: bool,
 }
 
 impl PlayerWrapper {
-    pub fn new(path: &Path) -> Self {
+    fn set_gl_params(player: Arc<Mutex<Box<Player>>>, context: &glutin::Context) -> bool {
```

Comment:
You might want to add some `dyn` here for trait objects. Not necessary but gives a better indication what the thing actually is.